pkg/apis/autopilot/v1alpha1/actionapproval.go

```diff
@@ -45,7 +45,7 @@ type ActionApprovalSpec struct {
 // ActionApprovalStatus is current status of an autopilot action approval
 type ActionApprovalStatus struct {
 	// Rule is the parent autopilot rule that resulted in this action approval
-	Rule types.NamespacedName
+	Rule types.NamespacedName `json:"rule"`
```

Comment:
Wouldn't this break backward compatibility? If we have an ActionApproval object already created which has the Status field set with `Rule`, the autopilot controller won't be able to read that object specifically the Rule filed.

Reply:
good point, let me test it first